recipes/objectpath/meta.yaml

```diff
@@ -0,0 +1,48 @@
+{% set name = "objectpath" %}
+{% set version = "0.6.1" %}
+
+package:
+  name: "{{ name|lower }}"
+  version: "{{ version }}"
+
+source:
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz
+  sha256: 461263136c79292e42431fbb85cdcaac4c6a256f6b1aa5b3ae9316e4965ad819
+
+build:
+  number: 0
+  entry_points:
+    - objectpath = objectpath.shell:main
+  script: "{{ PYTHON }} -m pip install . -vv "
+
+requirements:
+  host:
+    - pip
+    - python
+    - wheel
+  run:
+    - python
+    - pyreadline  # [win]
+    - readline    # [not win]
+    - pytz
+    - setuptools
+
+test:
+  imports:
+    - objectpath
+    - objectpath.core
+    - objectpath.utils
+  commands:
+    - objectpath --help
+
+about:
+  home: http://adriank.github.io/ObjectPath
+  license: MIT
+  license_family: MIT
+  license_file: LICENSE
+  summary: The agile query language for semi-structured data. #JSON
+
+extra:
+  recipe-maintainers:
+    - sp-hbk
+    - pg-hbk
```

Comment:
@pg-hbk - are you ok with being a maintainer?

Reply:
Yes.